src/java/OAC/sequence.java

```diff
@@ -2,236 +2,210 @@
+            query = QueryFactory.create(queryString);
+            qe = QueryExecutionFactory.create(query, m3Model);
+            results = qe.execSelect();
+            out.print("Query results:\n");
+            while (results.hasNext()) {
+                QuerySolution qs = results.next();
+
+                //just for testing the query
+                //System.out.print("sub:"+qs.get("subject").toString()+"\n");
+                out.print("Position:" + positionCounter + "\n");
+                out.print("Canvas:" + qs.get("predicate").toString() + "\n");
+                Resource r = qs.get("predicate").asResource();
+                //System.out.print(" stats:"+r.getId()+" :"+r.isAnon()+", "+r.isLiteral()+", "+r.isResource()+", "+r.isURIResource()+"\n");
+
+                //System.out.print(qs.get("predicate")+"\n");
+                Query innerQuery = QueryFactory.create("select  ?pred where { <" + r.getURI() + "> " + "<http://purl.org/dc/elements/1.1/title> ?pred}");
+                qe = create(innerQuery, m3Model);
+                ResultSet innerResults = qe.execSelect();
+                while (innerResults.hasNext()) {
+                    QuerySolution qs2 = innerResults.next();
+                    out.print("Title:" + qs2.get("pred") + "\n");
+
+                    Query innerQuery3 = QueryFactory.create("select ?pred  where { ?pred <http://www.openannotation.org/ns/hasTarget> <" + r.getURI() + "> }");
+                    qe = create(innerQuery3, sequenceModel);
+                    ResultSet innerResults2 = qe.execSelect();
+                    while (innerResults2.hasNext()) {
+                        QuerySolution innerqs2 = innerResults2.next();
+                        //System.out.print("1:"+innerqs2.get("pred")+" "+"\n");
+                        innerqs2.getResource("pred");
+
+                        //now get the image
+                        Query innerQuery4 = QueryFactory.create("select ?pred  where { <" + innerqs2.getResource("pred").getURI() + "> <http://www.openannotation.org/ns/hasBody> ?pred }");
+                        qe = create(innerQuery4, sequenceModel);
+                        ResultSet innerResults4 = qe.execSelect();
+                        if (innerResults4.hasNext()) {
+                            QuerySolution innerqs4 = innerResults4.next();
+                            out.print("Image:" + innerqs4.get("pred") + " " + "\n\n");
+                            positionCounter++;
+                        }
+                    }
                 }
-                }   
-                
-                
-                
-                
-                
-             
             }
-            
         }
-
-
+        if (true) {
+            return;
```

Comment:
Isn't this always `true`?  Doesn't this always `return`.  Should this be deleted...

Reply:
There are a many places like this where something had been pruned because OAC was updated or there was no more partner project QENI pipeline. I deleted about half, but I left the ones that I was not certain about (or overlooked). I should have ticketed them all. I will do that.

Reply:
There are a many places like this where something had been pruned because OAC was updated or there was no more partner project QENI pipeline. I deleted about half, but I left the ones that I was not certain about (or overlooked). I should have ticketed them all. I will do that.